scripts/zones/Windurst_Waters_[S]/npcs/Door_Rhinostery.lua

```diff
@@ -0,0 +1,50 @@
+-----------------------------------
+--  Area: Windurst Waters (S)
+--   NPC: Door
+-- @zone: 94
+
+-----------------------------------
+package.loaded["scripts/zones/Windurst_Waters_[S]/TextIDs"] = nil;
+-----------------------------------
+
+-----------------------------------
+-- onTrade Action
+-----------------------------------
+
+function onTrade(player,npc,trade)
+end;
+
+-----------------------------------
+-- onTrigger Action
+-----------------------------------
+
+function onTrigger(player,npc)
+	if (player:getQuestStatus(CRYSTAL_WAR, THE_LOST_BOOK)==QUEST_AVAILABLE and player:hasKeyItem(925)) then
+	player:startEvent(143);
+	player:addQuest(CRYSTAL_WAR, THE_LOST_BOOK); 
+	elseif player:hasKeyItem(971) then
+	player:startEvent(144);
+	elseif (player:getQuestStatus(CRYSTAL_WAR, THE_LOST_BOOK)==QUEST_ACCEPTED) then
+	end
```

Comment:
Yea I think I got a good grasp on it yesturday, Teo and I did some
teamviewer / anydesk and went over quite a lot. In my opinion this is all a
lot more complicated than it has to be and very strict and picky(not you
guys, the process) but I do understand it helps with cleanliness of the
code, performance, as well as when its pushed through and auto scanned it
saves you guys a lot of legwork looking for errors / mistakes. Hopefully I
can actually share something with quality work soon enough. I still have
recall quests to share, dark ixion, and some rune fencer stuff
(abiliites/rune enhancement etc). Started assaults currently as well, just
want enough time to get some work done here before I have to get back to a
busy schedule again.

On Sat, Dec 19, 2015 at 3:12 AM, Tahir Akhlaq <notifications@github.com>
wrote:

> In scripts/zones/Windurst_Waters_[S]/npcs/Door_Rhinostery.lua
> <https://github.com/DarkstarProject/darkstar/pull/2496#discussion_r48091419>
> :
>
> > +
> > +function onTrade(player,npc,trade)
> > +end;
> > +
> > +-----------------------------------
> > +-- onTrigger Action
> > +-----------------------------------
> > +
> > +function onTrigger(player,npc)
> > +	if (player:getQuestStatus(CRYSTAL_WAR, THE_LOST_BOOK)==QUEST_AVAILABLE and player:hasKeyItem(925)) then
> > +	player:startEvent(143);
> > +	player:addQuest(CRYSTAL_WAR, THE_LOST_BOOK);
> > +	elseif player:hasKeyItem(971) then
> > +	player:startEvent(144);
> > +	elseif (player:getQuestStatus(CRYSTAL_WAR, THE_LOST_BOOK)==QUEST_ACCEPTED) then
> > +	end
>
> 4 space indent for each item within control block (applies to control
> blocks within control blocks too)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DarkstarProject/darkstar/pull/2496/files#r48091419>.
>